DESCRIPTION

```diff
@@ -1,12 +1,34 @@
 Package: miaTime
 Type: Package
-Title: What the Package Does (Title Case)
+Title: Microbiome Time Series Analysis
 Version: 0.1.0
-Author: Who wrote it
-Maintainer: The package maintainer <yourself@somewhere.net>
-Description: More about what it does (maybe more than one line)
-    Use four spaces when indenting paragraphs within the Description.
-License: What license is it under?
+Authors@R:
+     c(person(given = "Leo", family = "Lahti", role = c("aut"),
+              email = "leo.lahti@iki.fi",
+              comment = c(ORCID = "0000-0001-5537-637X")),
+       person(given = "Yagmur", family = "Simsek", role = c("aut", "cre"),
+              email = "yagmur.simsek.98@gmail.com"))
+Description: 
+    Time series analysis of microbiome data sets is evaluated by the functions
+    in `miaTime` package.
+biocViews: Microbiome, Software, Sequencing, Coverage
+License: Artistic-2.0 | file LICENSE
+Depends:
+    SummarizedExperiment
+Imports:
+    methods,
+    TreeSummarizedExperiment
+Suggests: 
+    rmarkdown,
+    knitr,
+    mia,
+    devtools,
+    BiocStyle,
+    testthat (>= 3.0.0)
 Encoding: UTF-8
-LazyData: true
-RoxygenNote: 7.1.1
+URL: https://github.com/microbiome/miaTime
+LazyData: false
+Roxygen: list(markdown = TRUE)
+RoxygenNote: 7.1.2
+VignetteBuilder: knitr
+Config/testthat/edition: 3
```

Comment:
I would remove this not standard config info

Reply:
We also seem to have the first 3 of these in `mia` and `miaViz` DESCRIPTION files. Those to be removed as well?